waku/v2/protocol/waku_rln_relay/waku_rln_relay_metrics.nim

```diff
@@ -31,3 +42,48 @@ declarePublicGauge(waku_rln_registration_duration_seconds, "time taken to regist
 declarePublicGauge(waku_rln_instance_creation_duration_seconds, "time taken to create an rln instance")
 declarePublicGauge(waku_rln_membership_insertion_duration_seconds, "time taken to insert a new member into the local merkle tree")
 declarePublicGauge(waku_rln_membership_credentials_import_duration_seconds, "time taken to import membership credentials")
+
+template parseAndAccumulate(collector: Collector, cumulativeValue: float64): float64 =
+  let total = parseCollectorIntoF64(collector)
+  let freshCount = max(total - cumulativeValue, 0)
```

Comment:
Curious to know whether `total < cumulativeValue` can ever happen?

Reply:
It cannot. Fixed in [eb9101f](https://github.com/status-im/nwaku/pull/1262/commits/eb9101ff1b19478b500921cbb93433e2f140071c)